apm-server.yml

```diff
@@ -742,14 +742,15 @@ output.elasticsearch:
 # reporting is disabled by default.
 
 # Set to true to enable the monitoring reporter.
-#xpack.monitoring.enabled: false
-
-# Uncomment to send the metrics to Elasticsearch. Most settings from the
-# Elasticsearch output are accepted here as well. Any setting that is not set is
-# automatically inherited from the Elasticsearch output configuration, so if you
-# have the Elasticsearch output configured, you can simply uncomment the
-# following line.
-#xpack.monitoring.elasticsearch:
+#monitoring.enabled: false
```

Comment:
Sorry for the late response here. With the latest changes from libbeat, both options - `xpack.monitoring` and `monitoring` are supported, but `xpack.monitoring` will be deprecated from `7.1.0` on. Afaik we usually still document deprecated settings, so I suggest to document them both, but clearly mark the deprecated setting. 
Also, when describing the new setting, could you please ensure to fully remove `xpack` also from the header and the description.

Reply:
We do typically document deprecated settings, but I chose to follow Beats lead by removing it from yml files and docs: elastic/beats/pull/11677+ elastic/beats/pull/11678. We can document the deprecated settings, but it will mean adding more complexities to the docs as this is beats documentation - which isn't necessarily a bad thing - just want you to be aware.

Reply:
I'm asking to see if there's any rhyme or reason as to why xpack was left in the yml files, or if that was an oversight.

Reply:
Thanks for checking @bmorelli25. 

Up to you if you want to leave the yml+docs as they are, just wanted to ensure it wasn't removed by mistake and is streamlined with documenting other/future deprecated features and settings.

Reply:
Cool beans - I appreciate you bringing this up! I'm very curious to hear the thought process for why beats did this the way they did. Based on their answers, I'll figure out the path forward and determine what I need to or don't need to do.